### PR TITLE
[SONARMSBRU-114] Document configuration options

### PR DESCRIPTION
**Pull request by duncanp-sonar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -659,3 +662,126 @@ It is not necessary to uninstall the manually-installed version of the sonar-run
...
+
+
+<a name="configuringMSBuildRunnerAnchor">
+# Appendix 2: Configuring the MSBuild SonarQube Runner
+</a>
+
+## Contents
+
+- [Supplying additional analysis settings](#supplying-additional-analysis-settings)
+- [Classifying projects as test projects](#classifying-projects-as-test-projects)
```
Comment (dbolkensteyn) [inline README.md:674]: @duncanpMS can you remove this? We're going to update SonarLint to not trigger issues on test projects

**Comment by dbolkensteyn** [comment]
LGTM - it's good that you added this, the `/d:` switch for instance was really missing.

File: README.md (hunk 1 above)
Comment (duncanp-sonar) [inline README.md:674]: @dbolkensteyn this section will still be relevant, won't it? (test projects are treated differently, so users need to know how to make sure test projects are correctly identified)

Comment (dbolkensteyn) [inline README.md:674]: yes you're right @duncanpMS - I was mixing things

**Comment by duncanp-sonar** [comment]
Thanks @dbolkensteyn.

**Pull request merged**